### PR TITLE
Remove workflow result post when workflow is cancelled

### LINKED ISSUE #109: Do not post workflow results if the workflow is cancelled
We thought it would be useful but in the end it's a pain. Let's avoid it.

https://github.com/quarkusio/quarkus-bot/blob/main/src/main/java/io/quarkus/bot/AnalyzeWorkflowRunResults.java

### PR DESCRIPTION
**Pull request by Luke1432**
Fix #109

### CODE REVIEW

**Review by gsmet — CHANGES_REQUESTED** [review]
You haven't started your branch from the `main` branch but from your other branch. This makes everything mixed up :/.

File: src/main/java/io/quarkus/bot/AnalyzeWorkflowRunResults.java [hunk 1]
```diff
@@ -129,7 +129,7 @@ void analyzeWorkflowResults(@WorkflowRun.Completed GHEventPayload.WorkflowRun wo
                 artifactsAvailable,
                 checkRunOptional.orElse(null),
                 WorkflowConstants.MESSAGE_ID_ACTIVE);
-        if (!quarkusBotConfig.isDryRun()) {
+        if (!quarkusBotConfig.isDryRun() && (workflowRun.getConclusion() != Conclusion.CANCELLED)) {
```
Comment (gsmet) [inline src/main/java/io/quarkus/bot/AnalyzeWorkflowRunResults.java]: So you're doing all the work to generate the report and then you don't post it?

This check should be added at the top just after this one: https://github.com/quarkusio/quarkus-github-bot/pull/128/files#diff-934d6328af4061325fdd799d48620dee046978df99a47ef0f9e182218fdb30e7R69 .

**Comment by Luke1432** [comment]
> You haven't started your branch from the `main` branch but from your other branch. This makes everything mixed up :/.

Should I take this one out and start the branch from the main branch?

File: src/main/java/io/quarkus/bot/AnalyzeWorkflowRunResults.java (hunk 1 above)
Comment (Luke1432) [reply src/main/java/io/quarkus/bot/AnalyzeWorkflowRunResults.java]: Does the line right under this one not post the report?

**Review by gsmet — APPROVED** [review]

**Pull request merged**